### PR TITLE
Show original currency info in notes column

### PR DESCRIPTION
**Pull request by riceissa**
This also fixes a bug in the hover-over implementation for the original
currency info (in the amount column).

For #134.

### CODE REVIEW

File: access-portal/backend/donationNotes.inc [hunk 1]
```diff
@@ -35,6 +41,13 @@ if ($val = $row['donor_retrospective']) {
   array_push($notes_pre_key_val_pairs, sprintf("<strong>Donor retrospective of the donation:</strong> %s", trim(cleanNotes($val))));
 }
 $notes_key_val_pairs = array();
+if ($row['amount_original_currency']) {
```
Comment (vipulnaik) [inline access-portal/backend/donationNotes.inc:44]: Can we filter out the case where donor_amount_reason exists?

**Review by vipulnaik — CHANGES_REQUESTED** [review]
When the `notes` is empty, I still want the currency info key-value pair to be printed on a new line with the "Other notes" prefix; would this be feasible?

**Comment by riceissa** [comment]
> When the `notes` is empty, I still want the currency info key-value pair to be printed on a new line with the "Other notes" prefix; would this be feasible?

I think it already does this? The currency info gets added to `$notes_pre_key_val_string` or `$notes_key_val_string` depending on whether a `donor_amount_reason` is given, and both of these are independent of `notes`.

I did however have a bug where the currency info was sometimes being printed twice, which I have now fixed.

File: access-portal/backend/donationNotes.inc (hunk 1 above)
Comment (riceissa) [reply access-portal/backend/donationNotes.inc]: Did this now https://github.com/vipulnaik/donations/pull/135/commits/ecfbdd2672517edc65280b9fed3af495908599b6

**Review by vipulnaik — APPROVED** [review]
LGTM. I realized that fixing my other feedback needs a change not directly tied to #134, so I added it into the PR myself and shall merge now.

**Pull request merged**